src/neuroconv/tools/nwb_helpers.py

```diff
@@ -47,20 +50,35 @@ def get_default_nwbfile_metadata() -> DeepDict:
 
 def make_nwbfile_from_metadata(metadata: dict) -> NWBFile:
     """Make NWBFile from available metadata."""
-    metadata = dict_deep_update(get_default_nwbfile_metadata(), metadata)
+
+    # Validate metadata
+    schema_path = Path(__file__).resolve().parent.parent / "schemas/base_metadata_schema.json"
+    with open(schema_path) as schema_file:
+        base_metadata_schema = json.load(schema_file)
```

Comment:
We have a `util` that would take less lines and I think it might use the deocder implicitly; I think it's named `load_dict_from_file`?

Reply:
done